index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <title>Document</title>
+</head>
+<body id="root" class="container-fluid">
+    <!--Navbar-->
+    <nav class="">
+        <a class="logo" href="#">Welcome</a>
+        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-list" viewBox="0 0 16 16">
+            <path fill-rule="evenodd" d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"/>
+          </svg>
+    </nav>
```

Comment:
Nice job adding a navbar! Although, please consider adding a `header`. Kindly notice that the [header](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) element should contain your logo (it can be a text for now) and a [nav](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) element with your navigation links (in this case, your hamburger icon) 😉. By knowing this, please add a `header` to make your project semantically accurate and improve your page accessibility.   

You can have something like this:  
```html 
<header> 
  <a href=#>Welcome</a>  

  <nav> 
    Hamburger icon 
  </nav> 
</header> 
```